tests/lit-tests/2650.ispc

```diff
@@ -0,0 +1,12 @@
+// RUN: %{ispc} --target=host --nostdlib --nowrap -h %t.h %s 2>&1
+// RUN: FileCheck %s --input-file=%t.h
+// RUN: %{cc} %t.h
+
+// CHECK: struct uint32_t2
```

Comment:
Please add `// CHECK-LABEL: struct Test` before this check, otherwise it will be found in `struct uint32_t2` declaration itself.

Reply:
@aneshlya theoretically `struct Test` could _also_ be found in `extern void use(struct Test &test);` a few lines later, but I assume:

1. `CHECK-LABEL` will match on the first `struct Test` it finds (which will be the declaration here);
2. After `extern void use(struct Test &test);` `CHECK: struct uint32_t2` won't be found and would falsely fail rather than falsely succeed.


Perhaps it is more clear to write out `// CHECK struct uint32_t2 size;` fully?

Reply:
`CHECK-LABEL` should match the first occurrence. Anyway I agree that checking `// CHECK struct uint32_t2 size` would be even better.

Reply:
I check `struct uint32_t2 size` now.